students/11th/taesukim/project/project/urls.py

```diff
@@ -0,0 +1,20 @@
+"""project URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.0/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.urls import include, path
+
+urlpatterns = [
+    path('User/', include('User.urls')),
```

Comment:
엔드포인트주소는 대문자로 사용하지 않고 소문자로 사용합니다. 아래와 같이 고쳐주세요!
```suggestion
    path('user', include('User.urls')),
```